fast/stages/03-gke-multitenant/prod/gke-membership.tf

```diff
@@ -0,0 +1,99 @@
+
+locals {
+  config_management_wid_sa_email = (
+    var.config_management.workload_identity_sa == null
+    ? (
+      length(module.gke-wid-platform-sa[0]) > 0
+      ? module.gke-wid-platform-sa[0].email
+      : null
+    )
+    : var.config_management.workload_identity_sa
+  )
+  config_management_repository_url = (
+    var.config_management.repository_url == null
+    ? (
+      length(module.gke-config-management-platform-repo[0]) > 0
+      ? module.gke-config-management-platform-repo[0].url
+      : null
+    )
+    : var.config_management.repository_url
+  )
+}
+
+
+#add the cluster to the gke hub
+resource "google_gke_hub_membership" "membership" {
+  provider      = google-beta
+  for_each      = local.clusters
+  membership_id = each.key
+  project       = module.gke-project-0.project_id
+  endpoint {
+    gke_cluster {
+      resource_link = "//container.googleapis.com/projects/${module.gke-project-0.project_id}/locations/${each.value.location}/clusters/${each.key}"
+    }
+  }
+}
+#enable configmanagement feature on the the clusters registered into the hub
```

Comment:
- leave space between modules
- comments are not code but should use consistent style